index.js

```diff
@@ -0,0 +1,53 @@
+const booksArrays = [];
+let booksObject = {};
+let booksFromStorage = JSON.parse(localStorage.getItem('bookStore')) || [];
+
+//  function to delete book
+
+function deleteBook(button) {
+  const btnId = Number(button.getAttribute('data-id'));
+  const newBooks = booksFromStorage.filter((book) => book.id === btnId);
+  localStorage.setItem('bookStore', JSON.stringify(newBooks));
+  booksFromStorage = newBooks;
+}
+
+const container = document.getElementById('books');
+
+// function to display book
+
+function displayBook(booksObject) {
+  const bookAuthor = document.createElement('p');
+  const removeButton = document.createElement('button');
+  const horizontalLine = document.createElement('hr');
+  bookAuthor.innerText = `${booksObject.title} by ${booksObject.author}`;
+  removeButton.innerText = 'Remove';
+  removeButton.className = 'button';
+  removeButton.setAttribute('data-id', booksObject.id);
+  removeButton.addEventListener('click', () => {
+    deleteBook(removeButton);
+    container.removeChild(removeButton);
+    container.removeChild(bookAuthor);
+    container.removeChild(horizontalLine);
+  });
+  container.appendChild(bookAuthor);
+  container.appendChild(removeButton);
+  container.appendChild(horizontalLine);
+}
+
+// function to add book //
+
+// eslint-disable-next-line no-unused-vars
+function addBook(e) {
+  e.preventDefault();
+  const titleInput = document.getElementById('title').value;
+  const authorInput = document.getElementById('author').value;
+  const id = Math.floor(Math.random() * 1000);
+  booksObject = { author: authorInput, title: titleInput, id };
+  booksArrays.push(booksObject);
+  displayBook(booksObject);
+  localStorage.setItem('bookStore', JSON.stringify(booksArrays));
+}
```

Comment:
- [ ] Please refactor the localstorage configs in this add method because, in the browser, it doesn't actually add the book and display it with other books from the first attempt. 🙏

Reply:
@MahmoudBakr23 actually it is not clear to us because when we delete an item and we check in the browser its cleared and also removed from the UI. Could you please explain more on the change you want